src/commands/make-publish.ts

```diff
@@ -0,0 +1,35 @@
+import { cyan } from 'chalk';
+import { Command } from '@oclif/command';
+
+import { loadConfig } from '../config';
+import { printInfo, printLine } from '../util/io';
+import * as fileMakerService from '../service/fileMaker';
+
+class MakePublish extends Command {
+  static description = 'Publish migration templates files.';
+
+  /**
+   * CLI command execution handler.
+   *
+   * @returns {Promise<void>}
+   */
+  async run(): Promise<void> {
+    const config = await loadConfig();
+
+    await printLine('');
+
+    const templates = await fileMakerService.publish(config);
+
+    for (const template of templates) {
+      await printLine(cyan(`  - ${template}`));
```

Comment:
Should we print the full path of the template so that the user know where the template file is? I believe this is only the template name?

Reply:
Done.